### ISSUE TITLE
[Codelab]: Making Predictions from 2D Data

### ISSUE THREAD

**Issue by neerGGreen** [tensorflow/tfjs#2770]
I am attempting to run the tutorial from this [example](https://codelabs.developers.google.com/codelabs/tfjs-training-regression/index.html#1).  I've copied the code directly.  It seems to run but the predicted values output in the scatter plot are a constant value of 9.  The loss seems to be decreasing to an acceptable value, something that would not be possible if the model only is capable of output a Y-value of 9.  Any suggestions are appreciated.

[Tensorflow 2D Results.pdf](https://github.com/tensorflow/tfjs/files/4222911/Tensorflow.2D.Results.pdf)

**Comment by tafsiri** [comment]
@neerGGreen could you post a link to your implementation on something like codepen? Its otherwise hard to tell where the problem may be.

**Comment by rthadur** [comment]
Closing this due to lack of activity, feel to reopen. Thank you
